src/main/java/com/manufacturing/web/filters/AuthenticationFilter.java

```diff
@@ -0,0 +1,47 @@
+package com.manufacturing.web.filters;
+
+import com.manufacturing.lib.Injector;
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.Set;
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class AuthenticationFilter implements Filter {
+    private static final String DRIVER_ID = "driver_id";
+    private static final Injector injector = Injector.getInstance("com.manufacturing");
+    private final Set<String> urls = new HashSet<>();
```

Comment:
```suggestion
    private final Set<String> allowedUrls = new HashSet<>();
```
URLs name is a bit ambigours